hooks/useGame.js

```diff
@@ -0,0 +1,55 @@
+import { useState, useEffect } from "react";
+import { entitites } from "../components";
+import { physics } from "../utils";
+
+export const useGame = () => {
+  const [running, setRunning] = useState(false);
+  const [gameEngine, setGameEngine] = useState(null);
+  const [currentPoints, setCurrentPoints] = useState(0);
+  const entities = entitites();
+
```

Comment:
I removed your useEffect

 `  // useEffect(() => {
  //   setRunning(false);
  // }, []);` from this file (originally in the component). It wasn't doing anything, just providing an unnecessary rerender  as you start with running === false